core/src/main/java/hudson/model/BooleanParameterDefinition.java

```diff
@@ -72,7 +73,7 @@ public BooleanParameterValue getDefaultParameterValue() {
         return new BooleanParameterValue(getName(), defaultValue, getDescription());
     }
 
-    @Extension
+    @Extension @Symbol("booleanParam")
```

Comment:
Other than possible reserved-word clashes, I would think this would just be `boolean` (similarly `choice`, `file`, etc.).